go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/google/gofuzz v1.1.0
 	github.com/google/uuid v1.1.2
 	github.com/googleapis/gnostic v0.5.1
+	github.com/json-iterator/go v1.1.12 // indirect
```

Comment:
this is unfortunate (xref https://github.com/kubernetes/kube-openapi/pull/246)... is this actually used?